demisto_sdk/commands/validate/tests/DO_validators_test.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from demisto_sdk.commands.validate.tests.test_tools import (
+    create_integration_object,
+    create_script_object,
+)
+from demisto_sdk.commands.validate.validators.DO_validators.DO108_docker_image_exist import (
+    DockerImageExistValidator,
+)
+
+
+@pytest.mark.parametrize(
```

Comment:
to all of these test files we should add a few tests to make sure the message is as expected